dev-libs/libksba/libksba-1.3.5.recipe

```diff
@@ -0,0 +1,71 @@
+SUMMARY="Library for working with X.509 certificates, CMS data and related"
+DESCRIPTION="Libksba is a library to make the task of working with X.509, CMS \
+data and related objects easier. It provides a highlevel interface to \
+implemented protocols and presents the data in a consistent way."
+HOMEPAGE="https://www.gnupg.org/related_software/libksba/"
+COPYRIGHT="2001-2015 g10 Code GmbH"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://www.gnupg.org/ftp/gcrypt/libksba/libksba-$portVersion.tar.bz2"
+CHECKSUM_SHA256="41444fd7a6ff73a79ad9728f985e71c9ba8cd3e5e53358e70d5f066d35c1a340"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+PROVIDES="
+	libksba$secondaryArchSuffix = $portVersion
+	lib:libksba$secondaryArchSuffix = 8.11.6 compat >= 8
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libgpg_error$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libksba${secondaryArchSuffix}_devel = $portVersion
+	devel:libksba$secondaryArchSuffix = 8.11.6 compat >= 8
+	cmd:ksba_config$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_devel="
+	libksba$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libgpg_error$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:automake
+	cmd:awk
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	"
+
+BUILD()
+{
+	./autogen.sh
+	runConfigure ./configure
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+	rm $libDir/libksba.la
+
+	prepareInstalledDevelLib libksba
+	fixPkgconfig
+
+	packageEntries devel \
```

Comment:
The rm is only executed if $secondaryArchSuffix is not an empty string.
And since that variable is empty for any primary arch (and not empty otherwise) we have a simple method to keep the doc only once, i.e. in the hpkgs created for primary architectures.